applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -345,6 +345,16 @@ static int lc3_enable_cb(struct bt_audio_stream *stream, const struct bt_codec_d
 	ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_STREAMING);
 	ERR_CHK(ret);
 
+	/* MCS discover needs to be done one per connection */
+	if (IS_ENABLED(CONFIG_BT_MCC)) {
+		ret = ble_mcs_discover(stream->conn);
```

Comment:
This is now done even later than before, what happens if play/pause is pressed shortly after connecting?

Reply:
The previous solution never ensured that play/pause worked after connecting anyhow. This is basically just being pushed by 2-4 connection intervals, so it doesn't introduce any new issues that didn't exist before. 

The question is still valid though, and I cannot answer that (presumably the call to `bt_mcc` would just return an error as the handle to the control point characteristic would be unknown).